O365/message.py

```diff
@@ -756,6 +762,24 @@ def reply(self, to_all=True):
 
         message = response.json()
 
+        # adding new feacture to the timezone 
```

Comment:
This comment should be deleted, we can see the changes in the diff

Reply:
Looks like you figured it out, let's just keep working at this, we can see the commit history of this PR here: https://github.com/O365/python-o365/pull/1043/commits